servo

```diff
@@ -168,6 +169,20 @@ def describe():
 
     return d
 
+def environment(param):
+    print('verifying environment', param)
```

Comment:
Is this a debug message? Should it be treated as such (e.g. to stderr rather than stdout?)

Reply:
this is std.